sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/repository/TestRepositoryConfig.java

```diff
@@ -38,6 +38,9 @@ public class TestRepositoryConfig extends AbstractCosmosConfiguration {
     @Value("${cosmos.queryMetricsEnabled}")
     private boolean queryMetricsEnabled;
 
+    @Value("${cosmos.maxDegreeOfParallelism:0}")
```

Comment:
Same here, why do we need to mention the value next to the name of the property.

Reply:
Done.